ui/components/component-library/text-field-search/README.mdx

```diff
@@ -1,37 +1,47 @@
 import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
 
+import { TextFieldBase, TextField } from '..';
+
 import { TextFieldSearch } from './text-field-search';
 
 # TextFieldSearch
 
-The `TextFieldSearch` allows users to enter text to search. It wraps the `TextField` component that adds a search icon to the left of the input.
+The `TextFieldSearch` allows users to enter text to search
 
 <Canvas>
   <Story id="ui-components-component-library-text-field-search-text-field-search-stories-js--default-story" />
 </Canvas>
 
 ## Props
 
-The `TextFieldSearch` accepts all props below as well as all [Box](/docs/ui-components-ui-box-box-stories-js--default-story#props), [TextField](/docs/ui-components-component-library-text-field-text-field-stories-js--default-story#props) component props
+The `TextFieldSearch` accepts all props below as well as all [Box](/docs/ui-components-ui-box-box-stories-js--default-story#props) component props
 
 <ArgsTable of={TextFieldSearch} />
 
-### Show Clear Button
+`TextFieldSearch` accepts all [TextField](/docs/ui-components-component-library-text-field-text-field-stories-js--default-story#props)
+component props
+
+<ArgsTable of={TextField} />
+
+`TextFieldSearch` accepts all [TextFieldBase](/docs/ui-components-component-library-text-field-base-text-field-base-stories-js--default-story#props)
+component props
+
+<ArgsTable of={TextFieldBase} />
 
-Use the `showClearButton` prop to display a clear button when `TextFieldSearch` has a value. Use the `clearButtonOnClick` prop to pass an `onClick` event handler to clear the value of the input.
+### Clear Button On Click
```

Comment:
Changing this to reflect the prop updates. I think instead of exposing `showClearButton` and defaulting it to `true` it should just be apart of the child component with the ability to override using `...props`